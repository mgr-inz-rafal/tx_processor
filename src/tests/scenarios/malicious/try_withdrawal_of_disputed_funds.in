type,client,tx,amount
deposit,1,1,100.0
deposit,1,2,50.0
dispute,1,1,
withdrawal,1,3,100
