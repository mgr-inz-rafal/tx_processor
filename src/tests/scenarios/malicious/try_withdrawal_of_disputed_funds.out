client,available,held,total,locked
1,50,100,150,false